### ISSUE TITLE
VoiceOver: Alerts are often skipped

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#417]
In a screen share with Taliesin,  she showed me that VoiceOver is often skipping aria-live alerts. We observed it most frequently when releasing the balloon,  we don't get any movement or velocity alerts. We also observed it when the wall was added and removed from the play area.

Also, sometimes aria-live alerts just die altogether on VoiceOver.

This is most likely a performance problem, as the alerts are there on all other AT and in the a11y-view.

Testing on Bose (MacOS 10.13.3),  I haven't been able to reproduce this problem yet, Ill try a different device that may be slower.

**Comment by jessegreenberg** [comment]
Hmm, on Bose, every once and a while the alerts get quiet after the balloon is released, but I can still hear them.

**Comment by jessegreenberg** [comment]
I tested it out on Dirac (macOS 10.12), the QA team said this device is usually pretty slow. I am not able to reproduce this problem on that device either, the alerts are coming through as I would expect.

**Comment by jessegreenberg** [comment]
Testing on Jordan (MacBook Air, macOS 10.13.2). I played with the sime for ~7 minutes and never didn't observe the issue. But I found one way to reproduce with some consistency, (maybe alerts are ignored 60% of the time):
- Remove wall
- Get yellow balloon charged, drag it over to the right edge of play area
- Release balloon by *tabbing* to the "Add Wall" button.

In this case I sometimes don't get any alerts until the balloon hits the sweater.

**Comment by jessegreenberg** [comment]
Testing on Magnus (macOS 10.10.5), I played with the sim for about 8 minutes and did not encounter this bug at all.

**Comment by jessegreenberg** [comment]
@terracoda can you please verify that this was still a problem after restarting your laptop?

**Comment by jessegreenberg** [comment]
@lmulhall-phet could you also please try to reproduce this on your device? Let me know if you have any questions about the problem we are looking for.

**Comment by lmulhall-phet** [comment]
I think it might have to do with tabbing to the balloon right away.

Check out this video:

https://drive.google.com/open?id=1GwgIvgC8leeKjDLPKgMXQONdVoZ26uEo

Warning: RIP eardrums at the end. (Sorry.)

**Comment by jessegreenberg** [comment]
Awesome, thanks @lmulhall-phet - I actually didn't hear *any* alerts throughout that whole video, is that correct? Are you ever able to hear the alerts?

**Comment by lmulhall-phet** [comment]
I thought the movement alerts you were looking for were "up, down, left, right, etc." and the velocity alerts were "moves quickly to the left, etc."?

**Comment by jessegreenberg** [comment]
Correct, @lmulhall-phet, my mistake.

**Comment by jessegreenberg** [comment]
For @terracoda the alerts that were missing are "Released. Moves very quickly...", as well as others. For her, the alerts failed to trigger even with basic interaction. In your video it is like no alerts are announced until you stop moving the balloon?

**Comment by jessegreenberg** [comment]
I am updating Bose to 10.13.5, lets see if it shows up then.

**Comment by lmulhall-phet** [comment]
> In your video it is like no alerts are announced until you stop moving the balloon?

@jessegreenberg, movement alerts can occur while the balloon is still moving, so don't worry about this. I think VO just gets a lot of alerts backlogged and it gives the movement one after the other ones have stopped.

**Comment by jessegreenberg** [comment]
Thanks @lmulhall-phet - I haven't been able to  get the "backlog" of alerts on VO when testing yet.

**Comment by jessegreenberg** [comment]
@lmulhall-phet and I discussed, and we tried to reproduce this case. It sounds like one way to get aria-live alerts to fail is to quickly tab into the sim as shown in @lmulhall-phet's video https://github.com/phetsims/balloons-and-static-electricity/issues/417#issuecomment-396380038.

Although on @lmulhall-phet we never heard "Released", we weren't able to reproduce the case where the "Released. {{balloon}} moves {{movement}}." alerts didn't trigger at all.

**Comment by terracoda** [comment]
@jessegreenberg, I am getting more velocity alerts than I did this afternoon. It seems a bit random when I get them and when I do not. 

When a balloon moves right on a release (towards the wall), I almost always get a velocity in the alert.

When the balloon is moving slowly to the left, or slowly up, I seem less likely to get velocity included in the alerts. Sometimes I just get ending alert, or just movement with no velocity and ending alert, or sometimes nothing or safari busy.

However, I do get velocity alerts more now that I have nothing open except Safari.

@jessegreenberg, could it have anything to do with when your fixed #413? 

At some point in the fixing of #411, I thought I was hearing velocity phrases in more of the movement alerts, which seemed beneficial. For example, velocity was sounding good when I tested #411 at this comment point https://github.com/phetsims/balloons-and-static-electricity/issues/411#issuecomment-395163027

Maybe that helps a bit in finding when VO lightened up on when it reads out velocity information.
